src/js/jsx/sections/layers/LayerGroup.jsx

```diff
@@ -116,7 +136,7 @@ define(function (require, exports, module) {
                                 disabled={this.props.disabled}
                                 document={this.props.document}
                                 layer={layer}
-                                childNodes={layerNode.children}/>
+                                hasChildren={!!layerNode.children}/>
```

Comment:
Hmm, I don't see a way to check whether or not a `Layer` model has children (`layerNode.children` is an array of `LayerNode` instances), maybe I am missing something?